pipeline/scripts/analysis_pipeline.jl

```diff
@@ -17,50 +15,70 @@ include(srcdir("AnalysisPipeline.jl"));
 
 ## Other dependencies
 # Analogy: tar_option_set(...) in targets
-using .AnalysisPipeline
+# Add processes for parallel computing and ensure all have same
+# dependencies/environment
+using Distributed
+addprocs(1)
+
+@everywhere begin
+    using DrWatson
+    quickactivate(@__DIR__(), "Analysis pipeline")
+    include(srcdir("AnalysisPipeline.jl"))
+end
+
+@everywhere using .AnalysisPipeline
 
 ## Run the pipeline steps
 # Analogy: list(tar_targets...) followed by tar_make(...) in targets
 # This is an intermediate commit that runs but will be updated with a job
 # scheduler for these tasks in the next commit. Probably Dagger.jl
 
 # Default parameter values and plot true Rt
-default_gi_param_dict = default_gi_params()
-true_Rt = default_Rt()
-plt_Rt = plot_Rt(true_Rt)
-latent_models_dict = default_epiaware_models()
-latent_models_names = Dict(value => key for (key, value) in latent_models_dict)
-tspan = default_tspan()
-inference_method = default_inference_method()
+# @everywhere begin
+#     default_gi_param_dict = default_gi_params()
+#     true_Rt = default_Rt()
+#     plt_Rt = plot_Rt(true_Rt)
+#     latent_models_dict = default_epiaware_models()
+#     latent_models_names = Dict(value => key for (key, value) in latent_models_dict)
+#     tspan = default_tspan()
+#     inference_method = default_inference_method()
+# end
+
+default_gi_param_dict_thunk = Dagger.@spawn default_gi_params()
```

Comment:
I assume they have exported `@spawn` and so we need to write `Dagger.@spawn`? Otherwise it is a bit clunky?

Reply:
Yeah `@spawn` is used by `Threads` e.g. `Threads.@spawn` which is exposed from `Base.Threads`, i.e. it is available from `Main` for every Julia . 

I'm assuming that the style of naming `Dagger.@spawn` throughout the `Dagger` docs, which I've propagated here is due to this.